nyc_taxi/anaconda-project.yml

```
@@ -15,21 +15,20 @@ examples_config:
 user_fields: [examples_config]
 
 channels:
-- pyviz/label/dev
 - defaults
 
 packages: &pkgs
 - python=3.11.8
```

We prefer not to pin exactly the Python version, 3.11 should be enough.

```suggestion
- python=3.11
```